lxde-base/lxde-common/lxde-common-0.99.1.ebuild

```diff
@@ -6,12 +6,12 @@ EAPI="6"
 inherit eutils
 
 DESCRIPTION="LXDE Session default configuration files and nuoveXT2 iconset"
-HOMEPAGE="http://lxde.org/"
+HOMEPAGE="https://wiki.lxde.org/en/LXDE_Common"
 SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~alpha amd64 ~arm ~arm64 ~ppc x86 ~arm-linux ~x86-linux"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~x86 ~arm-linux ~x86-linux"
```

Comment:
Looks like you need to sync your branch. amd64, arm, and x86 are stable in tree.

Reply:
Ooops, things happen when splititng from an older PR.
Adjustment done

Reply:
See https://github.com/gentoo/gentoo/pull/6617